site/content/contribute/getting-started/test-servers.md

```diff
@@ -0,0 +1,34 @@
+---
+title: "Test Servers"
+heading: "Pull Request Test Servers"
+description: "Leveraging our Cloud infrastructure we can spin up full environments on demand to test code submitted in a PR."
+date: 2022-01-28T00:00:00-04:00
+weight: 7
+---
+
+As part of the pull request review process reviewers may need to test and verify the changes. Leveraging our Cloud infrastructure we can spin up full environments on demand to test code submitted in PRs.
+
+Core committers and staff can trigger test server creation on a PR by adding one of the following labels:
+
+* `Setup Cloud Test Server`: Triggers the creation of a standard test server using the latest commit on PR and a Postgres database.
+* `Setup HA Cloud Test Server`: Triggers the creation of a test server that has high availability.
+* `Setup Cloud + CWS Test Server`: Triggers the creation of a test server that connects to our test Customer Web Server.
+
+After adding these labels, a bot will comment on the PR that a test server is being created. It should take approximately 3-5 minutes for the server to create. Once it's ready the bot will comment on the PR again with a link and credentials to the test server for an admin and regular user.
+
+If the bot comments that an error has occurred, try removing the label and then re-adding it again. If that still fails please ask for help in [~Developers: Cloud](https://community.mattermost.com/core/channels/cloud). If you need urgent help, please mention `@sresupport` in your message.
+
+Once testing is complete, remove the label and the test server will be destroyed.
+
+Test servers are available on any repositories that have the labels.
+
+# Tips & Tricks
+
+* Avoid adding and removing the labels quickly in succession - this can confuse the bot and result in issues - please be patient :)
```

Comment:
```suggestion
* Avoid adding and removing the labels quickly in succession - this can confuse the bot and result in issues. Please be patient. :)
```